core/templates/pages/exploration-editor-page/preview-tab/preview-tab.component.ts

```diff
@@ -190,7 +189,7 @@ angular.module('oppia').component('previewTab', {
 
           // Prompt user to enter any unset parameters, then populate
           // exploration.
-          manualParamChanges = ctrl.getManualParamChanges(
```

Comment:
The plugin also catches variables like these -- where we always assign a value to a variable and it is never used, right?

Reply:
yes, right.